src/content/docs/contributing/nf_core_basic_training.md

```diff
@@ -681,19 +685,86 @@ INFO     Use the following statement to include this module:
  include { SALMON_INDEX } from '../modules/nf-core/salmon/index/main'
 ```
 
+The module is now installed into the folder `modules/nf-core`. Now open the file `workflow/demotest.nf`. You will find already several `include` statements there from the installed modules (`MultiQC` and `FastQC`):
+
+```bash
+
+include { FASTQC  } from '../modules/nf-core/fastqc/main'
+include { MULTIQC } from '../modules/nf-core/multiqc/main'
+```
+
+Now add the above line underneath it:
+
+```bash
+
+include { FASTQC  } from '../modules/nf-core/fastqc/main'
+include { MULTIQC } from '../modules/nf-core/multiqc/main'
+include { SALMON_INDEX } from '../modules/nf-core/salmon/index/main'
+
+```
+
+This makes the module now available in the workflow script and it can be called with the right input data.
+
+<!-- TODO/TO DISCUSS here the user now needs to know about how to get their fasta. We could do this here or add a new point for this above -->
+
+We can now call the module in our workflow. Let's place it after FastQC:
+
+```bash
+
+workflow DEMOTEST {
+
+    ...
+    //
+    // MODULE: Run FastQC
+    //
+    FASTQC (
+        INPUT_CHECK.out.reads
+    )
+    ch_versions = ch_versions.mix(FASTQC.out.versions.first())
+
+    SALMON_INDEX()
+```
+
+Now we are still missing an input for our module. In order to build an index, we require the reference fasta. Luckily, the template pipeline has this already all configured, and we can access it by just using `params.fasta` and `view` it to  insppect the channel content. (We will see later how to add more input files.)
```

Comment:
Could we add a section where we show how to determine which input channels are required?